test/README

```diff
@@ -10,7 +10,7 @@ References:
 	- http://en.wikipedia.org/wiki/Test_Anything_Protocol
 
 What is in a number?
-	0*	Test the test system itself - psudo tests
+	0*	Test the test system itself - pseudo tests
```

Comment:
Sneaking in a typo fix in an unrelated commit?

Reply:
Well, sort of, yes, my apologies. This was originally not intended for public release and I just cherry-picked the relevant commit and did not check it again. Is this ok, though?

Reply:
Typo fixes are always welcome - a patch is worth merging if the project is better after that. It is just confusing to review a patch if it contains several unrelated things.